src/components/common/datagrid.js

```diff
@@ -90,9 +90,13 @@ const StyledCell = styled.div`
   height: 100%;
 `;
 
-const CellTemplate = ({name, type}) => {
-  const value = type === ALL_FIELD_TYPES.boolean ? String(name) : name;
-  return (<StyledCell title={value}><span>{value}</span></StyledCell>);
+const CellTemplate = ({value, type}) => {
+  const content = type === ALL_FIELD_TYPES.boolean ? String(value) : value;
+  return (
+    <StyledCell title={content}>
+      <span>{content}</span>
+    </StyledCell>
+  );
 };
 
 export const CellFactory = () => CellTemplate;
```

Comment:
this could cause compile and uglify error

```js
export const CellFactory = () => {
const CellTemplate = ({value, type}) => {}
return CellTemplate
}

```

Reply:
fixed